docs/Userguide_running_code.rst

```diff
@@ -350,33 +368,22 @@ Special GPU requirements
 Specific GPU *architecture* and *memory* can be easily requested through the
 ``--gres`` flag by using either
 
-* ``--gres=gpu:architecture:memory:number``
```

Comment:
Did we remove that? How can one request an A100 40GB vs. an A100 80GB?

Reply:
@btravouillon SLURM removed it. You can't supply a 4-part spec like that, it's rejected by SLURM's pre-parser as an invalid GRES spec before `job_submit.lua` can even get its paws on it. So I removed support for it from there long ago, as it is unreachable.